application-analytics-ui/src/main/resources/Analytics/WebHome.xml

```diff
@@ -199,9 +340,9 @@
       <name>platform.panels.AnalyticsApplication</name>
     </property>
     <property>
-      <parameters>label=$services.localization.render('analytics.extension.title')
-target=Analytics.WebHome
-icon=icon:chart-bar</parameters>
+      <parameters>label=$escapetool.xml($services.localization.render('analytics.extension.title'))
```

Comment:
No need to do this here as well, since it's escaped later on https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-application/xwiki-platform-application-ui/src/main/resources/Applications/Macros.xml#L50